spring-batch-docs/asciidoc/spring-batch-integration.adoc

```diff
@@ -1261,3 +1261,92 @@ You must also ensure that the partition `handler` attribute maps to the `partiti
 
 You can find a complete example of a remote partitioning job
 link:$$https://github.com/spring-projects/spring-batch/tree/master/spring-batch-samples#remote-partitioning-sample$$[here].
+
+The `@EnableBatchIntegration` annotation that can be used to simplify a remote
+ partitioning setup. This annotation provides two beans useful for remote partitioning:
+
+* `RemotePartitioningMasterStepBuilderFactory`: used to configure the master step
+* `RemotePartitioningWorkerStepBuilderFactory`: used to configure the worker step
+
+These APIs will take care of configuring a number of components as described in the following diagram:
```

Comment:
In fixes for Spring Integration docs I've seen that @Buzzardo changed all our future tenses to the present simple.
WDYT?

Reply:
Fixed. Thanks!

Reply:
Well if you ask me `WDYT?`, I need to tell you what I think right? 😜 But I've got the point, fair enough.